view/src/test/java/cmm/apps/esmorga/view/screenshot/eventdetails/EventDetailsScreenshotTest.kt

```diff
@@ -22,20 +22,20 @@ class EventDetailsScreenshotTest : BaseScreenshotTest() {
 
     @Test
     fun eventDetailsView_lightTheme_data_user_event_joined() {
-        snapshotWithState(userJoined = true, isAuthenticate = true)
+        snapshotWithState(buttonTitle = R.string.button_leave_event)
```

Comment:
Do not user R here, set any static text. If you use R here, when any value there changes, your screenshots will change and therefor your test will fail.